src/app/pages/candidate-details/candidate-details.analytics.effects.ts

```diff
@@ -21,6 +22,7 @@ import {
   isCandidateSpecialNeeds,
 } from '@store/candidate-details/candidate-details.selector';
 import { AnalyticRecorded } from '@providers/analytics/analytics.actions';
+import { mapAnalyticsScreenName } from '@shared/helpers/map-analytics-screen-name';
 
 @Injectable()
```

Comment:
Assume theres no spec file so can we add a TODO here indicating we might want to add one in future

Reply:
of course there was a test for it 🥇